### PR TITLE
Add command-line option to write verification summary to a file

### PR DESCRIPTION
**Pull request by nano-o**
This adds a command-line option `-s filename` which causes SAW to write the verification summary to the given file. Btw, I'm a Haskell novice and I would welcome constructive criticism.

### CODE REVIEW

File: src/SAWScript/Interpreter.hs
Comment (andreistefanescu) [inline src/SAWScript/Interpreter.hs:363]: I expect `writeFile` to work with both absolute and relative paths. Is that not the case?

File: src/SAWScript/Options.hs
Comment (andreistefanescu) [inline src/SAWScript/Options.hs:34]: nit: spacing

File: src/SAWScript/Interpreter.hs
Comment (nano-o) [reply src/SAWScript/Interpreter.hs:363]: Yes, but the current directory is set to the one in which the `.saw` file being process resides (in `processFile` in `Interpreter.hs`). I think that the user would instead expect the file to be written in the directory from which the command was invoked.

File: src/SAWScript/Options.hs
Comment (nano-o) [reply src/SAWScript/Options.hs]: fixed

**Review by jared-w — APPROVED** [review]
I was able to test out the `-s` behavior in the context of where we'd like to start using it. It works well, so if nobody has any objections to the code, it'd be great to merge this in sooner rather than later.

**Review by andreistefanescu — APPROVED** [review]
Looks good!

**Comment by nano-o** [comment]
@jared-w @andreistefanescu I can't merge it myself (I don't have write access I suppose).

**Comment by atomb** [comment]
I just added you to the `program-analysis` group, @nano-o, so you should be able to merge once CI finishes.

**Pull request merged**
